ports/oatpp-openssl/portfile.cmake

```
@@ -0,0 +1,27 @@
+set(OATPP_VERSION "1.2.5")
+
+vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
+
+# get the source
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO oatpp/oatpp-openssl
+    REF ${OATPP_VERSION}
+    SHA512 c5a40691d846703378c2965f2ad9b99edb2025d47854719bab4e80a4771258c2b72cfba0135b63dd9e2387da4549b5bc6cebfc1913a8006d14d59e250be19060
+    HEAD_REF master
+)
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+    PREFER_NINJA
+    OPTIONS
+        "-DOATPP_BUILD_TESTS:BOOL=OFF"
+        "-DCMAKE_CXX_FLAGS=-D_CRT_SECURE_NO_WARNINGS"
+)
+
+vcpkg_cmake_install()
+vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/oatpp-openssl-${OATPP_VERSION})
```

```suggestion
vcpkg_cmake_config_fixup(CONFIG_PATH "lib/cmake/oatpp-openssl-${OATPP_VERSION}")
```
We suggest to add quotes to path. Please see the [doc ](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/cmake-guidelines.md).

---

done